source/options.html

```diff
@@ -23,6 +23,13 @@ <h2 class="only-firefox">Options</h2>
 		</label>
 	</p>
 
+	<p>
+		<label>
+			<strong>Personal token</strong>, <a href="https://github.com/settings/tokens/new?description=Refined%20GitHub">found here</a> (for high rate API calls and private repos)<br>
+			<input type="password" name="personalToken" autocomplete="off" size="40" maxlength="40" pattern="[a-z0-9]{40}"></input>
```

Comment:
`type="password"` should be dropped, we shouldn't give the impression that the token is stored securely, because it isn't. No `type` is required